README.md

```diff
@@ -21,3 +25,5 @@ A sample iOS app supporting [XCUI Tests](https://developer.apple.com/library/con
 	5. zip -r SampleUITests.zip SampleXCUITests-Runner.app/
 5. Use the ipa, and zip file to run on Browserstack
 
+## Note : Follow [This](https://www.browserstack.com/docs/app-automate/xcuitest/getting-started) for executing the tests on BrowserStack
+
```

Comment:
Include a section Links & Resources in Readme and provide the link https://github.com/rwapp/A11yUITests & https://developer.apple.com/documentation/xctest